drools-benchmarks-parent/drools-benchmarks-reliability/src/main/java/org/drools/benchmarks/reliability/AbstractReliabilityBenchmark.java

```diff
@@ -67,6 +67,9 @@ public String getInfinispanStorageMode() {
     @Param({"true", "false"})
     private boolean useSafepoints;
 
+    @Param({"true", "false"})
+    private boolean useObjectStoreWithReferences;
+
```

Comment:
@nprentza Could you move this param to `FireAndAlarmBenchmark.java`?  I think we want to test this option only with `FireAndAlarmBenchmark` at the moment.

Reply:
@tkobayas yes, I will move the param.

Reply:
Please check commit [move useObjectStoreWithReferenceParam](https://github.com/kiegroup/kie-benchmarks/pull/256/commits/0c7f110f2a1ed6e56f819902e1c5ed86e0c9786e)

Reply:
Yeah, overriding the setup() method is fine. Thanks.

> How about a new AbstractReliabilityBenchmark abstract class for opt-int object store with references benchmarks, instead?

If we will have multiple benchmark tests with OBJECT_REFERENCES in the future, having a parent class would make sense, but at the moment, we don't need to do that.